bears/go/GoMetaLintBear.py

```diff
@@ -0,0 +1,86 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='gometalinter',
+        output_format='regex',
+        output_regex=r'\w+\.go:'
+                     r'(?P<line>\d+):'
+                     r'(?P<column>\d*):'
+                     r'(?P<severity>[a-z]+): '
+                     r'(?P<message>.*) '
+                     r'(?P<additional_info>\([a-z]+\))',
+        severity_map={'error': RESULT_SEVERITY.MAJOR,
+                      'warning': RESULT_SEVERITY.NORMAL})
+class GoMetaLintBear:
+    """
+    Lints your Go files!
+    Concurrently runs a number of Go lint tools.
+    """
+
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(
```

Comment:
put GoRequirement on the next line, indented.

Reply:
all of the Go bears follow this; still want me to change it?